### PR TITLE
Moved Cognito trigger here

### PR DESCRIPTION
**Pull request by janbj**
Java code is copied from another repository.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -3,47 +3,65 @@ plugins {
     id 'jacoco'
     id 'checkstyle'
     id 'pmd'
+    id 'nebula.lint' version '16.8.0'
```
Comment (axthosarouris) [inline build.gradle:6]: 16.23 is the latest

File: build.gradle [hunk 2]
```diff
@@ -3,47 +3,65 @@ plugins {
...
+        implementation group: 'org.slf4j', name: 'slf4j-api', version: '1.8.0-beta4'
+        implementation group: 'com.amazonaws', name: 'aws-lambda-java-core', version: '1.2.1'
+        implementation group: 'com.amazonaws', name: 'aws-java-sdk-lambda', version: project.ext.awsSdkVersion
+        implementation group: 'com.amazonaws', name: 'aws-java-sdk-cognitoidp', version: project.ext.awsSdkVersion
+        implementation group: 'org.apache.httpcomponents', name: 'httpclient', version: '4.5.9'
+        implementation group: 'org.zalando', name: 'problem', version: '0.25.0'
+
+        implementation group: 'com.github.BIBSYSDEV.nva-user-access-service', name: 'user-access-public-model', version: project.ext.nvaUserAccessServiceVersion
+        implementation group: 'com.github.BIBSYSDEV.nva-user-access-service', name: 'user-access-commons', version: project.ext.nvaUserAccessServiceVersion
+        implementation group: 'com.github.BIBSYSDEV.nva-user-access-service', name: 'user-access-errors', version: project.ext.nvaUserAccessServiceVersion
```
Comment (axthosarouris) [inline build.gradle:58]: Suggstion: add a TODO as reminder that these dependencies will have to be removed when we include the access service here

File: cognito-pre-token-generation/src/main/java/no/unit/nva/cognito/TriggerHandler.java [hunk 3]
```diff
@@ -0,0 +1,245 @@
+package no.unit.nva.cognito;
```
Comment (axthosarouris) [inline cognito-pre-token-generation/src/main/java/no/unit/nva/cognito/TriggerHandler.java:1]: I assume this code is copy paste and therefore I do not check it

**Review by axthosarouris — APPROVED** [review]
I did not go through the code because I assume it is a copy paste from the old project

**Review by brinxmat — APPROVED** [review]

**Review by svenngjeto — APPROVED** [review]

**Pull request merged**